File: LogicaDeProgramacaoEclipse(Java)/LogicaProgramacaoJava(Eclipse)/exerciciosJava/src/exerciciosFaccat/Exercicio14.java
package exerciciosFaccat;

import java.util.Scanner;

/*14) Ler um valor e escrever a mensagem É MAIOR QUE 10! se o valor lido for maior que 10, caso
contrário escrever NÃO É MAIOR QUE 10! */

public class Exercicio14 {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);

		System.out.println("Digite um valor para validar se é maior que 10 ou não:");
		float valor = sc.nextFloat();

		if (valor > 10) {
			System.out.println("O valor " + valor + " é maior que 10");
		}

		if (valor == 10) {
			System.out.println("O valor é igual a " + valor);
		}

		if (valor < 10) {
			System.out.println("O valor " + valor + " é menor que 10");
		}
		
		sc.close();
	}
}

File: LogicaDeProgramacao(JavaScript)/Exercicios 5 até o 39 Apostila Faccat/js/exercicio14.js
/*14) Ler um valor e escrever a mensagem É MAIOR QUE 10! se o valor lido for maior que 10, caso
contrário escrever NÃO É MAIOR QUE 10! */

var numero = parseInt(prompt("Digite um numero: "))

if (numero > 10) {
    alert("Numero digitado é maior que 10")
} else if (numero < 10) {
    alert("Numero digitado é menor que 10")
} else (
    alert("Numero digitado é igual a 10")
)